app/scripts/metamask-controller.js

```diff
@@ -2319,6 +2319,7 @@ export default class MetamaskController extends EventEmitter {
         allowedActions: [],
         allowedEvents: [],
       }),
+      fetchInterval: 15 * 60, // 15 minutes
```

Comment:
I believe this value needs to be set in milliseconds, [see the controller code](https://github.com/MetaMask/core/blob/main/packages/remote-feature-flag-controller/src/remote-feature-flag-controller.ts#L22C14-L22C36)

Reply:
🕵🏻‍♂️ good catch ! addressed bcc638b64cb5b46ad7102ef9bbdd2f999480c9fe